packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/translations/Messages_en_GB.json

```diff
@@ -1,8 +1,8 @@
 {
-  "dedicatedCloud_configuration_add_datacenter_title": "Add a datacentre",
-  "dedicatedCloud_configuration_add_datacenter_confirmation": "Please select the type of datacentre that you want to add:",
-  "dedicatedCloud_datacenters_adding_success": "A new datacentre will be created in a few moments.",
-  "dedicatedCloud_datacenters_adding_error": "An error has occurred adding a datacentre.",
-  "dedicatedCloud_datacenters_adding_load_error": "An error has occurred loading the information for the datacentre addition.",
+  "dedicatedCloud_configuration_add_datacenter_title": "Add a datacenter",
```

Comment:
We shouldn't update translations manually, if we want to change the content we should have a new translation request
Also `datacentre` is valid for `en_GB` locale

Reply:
The requirement was to change `datacentre` to `datacenter` in GB alone. There is no change on the FR side. This is a typo fix. I am not sure as to how we can raise a translation request when there is no change in the FR side. I thought fixing the typo is the best solution here. Please suggest

Reply:
@ganeshkumar1989 we should resubmit translations and only ask GB to be retranslated with the mention of the typo even though for en_GB it is not a typo (it is for en_US but not for GB)

Reply:
Just confirmed with the PU that this change is not required. Hence all the typo changes have been reverted. Thanks for your inputs @marie-j